scripts/obfuscator/path-to-obfuscated-key

```diff
@@ -0,0 +1 @@
+/Users/kirpichonok/Desktop/api-key
```

Comment:
Do we need this file?

Reply:
I updated this file. We use one to specify a path where the obfuscated key is stored. A script in the build phase will be able to read that path as well. That allows us to specify a path to the key in one place.

Reply:
Fixed.